dev/App.tsx

```diff
@@ -1,15 +1,16 @@
-import React, { Component } from 'react';
+import * as React from 'react';
+
 import { Widget, addResponseMessage, setQuickButtons, toggleMsgLoader } from '../index';
 
-export default class App extends Component {
+export default class App extends React.Component {
   componentDidMount() {
     addResponseMessage('Welcome to this awesome chat!');
   }
 
-  handleNewUserMessage = (newMessage) => {    
+  handleNewUserMessage = (newMessage: any) => {
```

Comment:
can't we define an interface for this? It seems like it's a string, but if it can be an object you can do 
```
type Message = string | { ... }
```

Reply:
Oh yeah, of course. But this is the `dev environment` so I will fix this after the migration is done 😄